### PR TITLE
Update roadmap.md

### PR DESCRIPTION
**Pull request by lemmaa**
Issue: #246

- Initial draft for this year's goal.

Signed-off-by: Sung-Jae Lee <sj925.lee@samsung.com>

### CODE REVIEW

File: docs/overview/roadmap.md [hunk 1]
```diff
@@ -1 +1,58 @@
...
+Until last year, we already saw significant gains in accelerating with a single CPU or GPU backend.
+We have seen better performance improvements, not only when using a single backend, but even when
+mixing CPUs or GPUs considering the characteristics of individual operations. It could give us an
+opportunity to have a high degree of freedom in terms of operator coverage, and possibly provide
+better performance compared to single backend acceleration.
+
+On the other hand, we introduced the compiler as a front-end. This will support a variety of deep
+learning frameworks in relatively spacious host PC environments, while the runtime running on the
+target device is intended to take a smaller burden. In this approach, the compiler and the runtime
+will effectively share information among themselves by the Common IR, named _circle_, and a
```
Comment (seanshpark) [inline docs/overview/roadmap.md:20]: _circle_ is all in lower cases. I used _Circle_  with capital letter _C_ to denote it's not just a circle (round object) but a proper noun. What do you think of using _Circle_ ?

File: docs/overview/roadmap.md [hunk 2]
```diff
@@ -1 +1,58 @@
...
+## Deliverables
+
+- Runtime
+  + Control Flow support (IF/WHILE)
+  + Dynamic Tensor support
+  + High quality kernel development for UINT8 quantized model
+  + End-to-end performance optimization for voice models
+- Compiler
+  + More than 100 operations support
+  + Standalone _circle_ interpreter
```
Comment (seanshpark) [inline docs/overview/roadmap.md:43]: ditto

File: docs/overview/roadmap.md [hunk 3]
```diff
@@ -1 +1,58 @@
...
+
+## Milestones
+
+- [2020 Project Milestones](https://github.com/Samsung/ONE/projects/1)
+
+## Workgroups (WGs)
+
+- We organize WGs for major topics, and each WG will be working on its own major topic by breaking
+  it into small tasks/issues, performing them inside WG, and collaborating between WGs.
+- The WG information can be found [here](workgroups.md).
```
Comment (llFreetimell) [inline docs/overview/roadmap.md:57]: Maybe `workgroups.md` --> `workgroup.md`? :)

File: docs/overview/roadmap.md [hunk 4]
```diff
@@ -1 +1,58 @@
...
+  + Dynamic Tensor support
+  + High quality kernel development for UINT8 quantized model
+  + End-to-end performance optimization for voice models
+- Compiler
+  + More than 100 operations support
+  + Standalone _circle_ interpreter
+  + Completion and application of _circle2circle_ pass
+    - _circle-quantizer_ for UINT8 and INT16
+    - _circle-optimizer_
+  + Grphical _circle_ model viewer
```
Comment (seanshpark) [inline docs/overview/roadmap.md:47]: ditto

File: docs/overview/roadmap.md (hunk 1 above)
Comment (lemmaa) [reply docs/overview/roadmap.md:20]: @seanshpark , from a long time ago, I generally used _itallic_ for all of our proper nouns. _NN Package_, _nnfw_, _nncc_, etc. for example. Only **ONE** uses the bold boldface, recently. Also, I prefer to express everything in lowercase letters whenever possible. We do not have explicit guidance on this. This is one of the important documents to be added.

If you aren't against it, what if all the documents I write are unified according to the rules above? If the guidelines are confirmed afterwards, it is good to reflect them in a batch.

File: docs/overview/roadmap.md (hunk 3 above)
Comment (lemmaa) [reply docs/overview/roadmap.md:57]: @llFreetimell , thanks!

File: docs/overview/roadmap.md (hunk 1 above)
Comment (seanshpark) [reply docs/overview/roadmap.md:20]: OK, sounds good :)

**Comment by seanshpark** [comment]
Just a question for `circle`. In source codes I can't use italic. just `circle` willbe OK as we usually don't talk about round shape in a diagram... 
But there is a network viewer program to develop... it may sound confused...

**Comment by lemmaa** [comment]
> But there is a network viewer program to develop... it may sound confused...

Actually, it's not hard goal of this year, but we need to deliver it for SDK for something next year. :)

**Pull request merged**